bindings.xjb

```diff
@@ -0,0 +1,58 @@
+<jxb:bindings version="2.1"
```

Comment:
This needs a license header.

Also, is this generated? Can this just be created as part of the build process?

Reply:
This is a manually generated file. Which license header should be added?

Reply:
ALv2